storybookjs/storybook
# CLI: Fix copyTemplate failures on `init`

Closes #22348 

## What I did

Add guards against projectType being an unknown value
Add guards against templateDir being a location that doesn't exist

## How to test

I'm not exactly sure how to reproduce the error, but 1 way that would work for sure would be to set a `projectType` to a odd value in an angular project. I don't think there are any other proper values other then `application` and `library` but if there are, then that would have caused issues in the past. After this PR that should default back to `application`. I assume this is the correct default.

If the templateDir cannot be found at all, we'll just skip it, this MIGHT cause those storybook to fail to start or build, but that's hard to tell without a full reproduction I can follow.
...either way that would become a follow-up issue then.
